modules/programs/mpv.nix

```diff
@@ -49,30 +49,28 @@ let
   renderDefaultProfiles = profiles:
     renderOptions { profile = concatStringsSep "," profiles; };
 
-  mpvPackage = if cfg.scripts == [ ] then
-    pkgs.mpv
-  else
-    pkgs.wrapMpv pkgs.mpv-unwrapped { scripts = cfg.scripts; };
-
 in {
+  imports = [
+    (mkRemovedOptionModule [ "programs" "mpv" "scripts" ] ''
+      The `scripts` part was removed in favor of using `package` option
+      with `pkgs.wrapMpv`.
+
+      Replace `programs.mpv.scripts = [ ... ]` with
+      `programs.mpv.scripts = pkgs.wrapMpv pkgs.mpv-unwrapped { scripts = [ ... ]; }`.
```

Comment:
Well, I took this from another module: https://github.com/thiagokokada/home-manager/blob/mpv-module-improvements/modules/programs/info.nix#L35-L38

But sure, can fix it.

Reply:
BTW I don't think this shows in manual, instead it shows in terminal if the user tries to use the removed command.

So I don't think any formatting applies, neither Markdown or Docbook. This is mostly to guide the user, but I could substitute this with `"` maybe (since I think this is more common).

Reply:
My mistake, you are right about the documentation.

https://github.com/NixOS/nixpkgs/blob/9e6737710c4fb2613850e699178b23d54f1a3261/lib/modules.nix#L892-L917

Let's go for the solution proposed below instead.

Reply:
Closing this since there is no more removed options.